made needed copy updates

mtruj013 opened the pull request:
## Done

Made needed copy updates, d3 chart still needs updating 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://localhost:8001/about/release-cycle
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]


## Issue / Card

Fixes #
https://github.com/canonical-web-and-design/ubuntu.com/issues/10219

## Screenshots

[If relevant, please include a screenshot.]

webteam-app commented:
Demo starting at https://ubuntu-com-10231.demos.haus

carkod commented on templates/about/release-cycle.html:
```diff
@@ -578,15 +578,15 @@ <h2 id="ubuntu-openstack-release-cycle">
 
     <div class="u-fixed-width">
-      <p>For more information on previous and upcoming Ubuntu OpenStack releases please see the <a href="https://wiki.ubuntu.com/OpenStack/CloudArchive " class="p-link--external ">Ubuntu Cloud Archive wiki page</a>.</p>
+      <p>For more information on supported versions, see <a href="https://ubuntu.com/openstack/docs/supported-versions" class="p-link--external ">Charmed OpenStack documentation</a>.</p>
```
This is no longer an external link, so please change it to relative path `/openstack/docs/supported-versions` and remove the `class="p-link--external "`

mtruj013 replied:
Updated, should work now

carkod commented on templates/about/release-cycle.html:
```diff
@@ -578,15 +578,15 @@ <h2 id="ubuntu-openstack-release-cycle">
       <div class="col-8">
-        <h2 id="charmed-kubernetes-release-cycle">Charmed Kubernetes<sup>&reg;</sup> release cycle</h2>
-        <p>The release cycle of the Charmed Kubernetes is tightly synchronised to the release of upstream Kubernetes<sup>&reg;</sup>. The current release and two prior releases are supported, giving an effective support period of nine months, subject to changes in the upstream release cycle.</p>
+        <h2 id="canonical-kubernetes-release-cycle">Canonical Kubernetes<sup>&reg;</sup> release cycle</h2>
```
You changed this id, but you didn't update the `href` in line 45, so that link is broken, can you please update that `href`?

carkod replied:
Is Canonical Kubernetes a registered trademark? You might need to remove that `<sup>&reg;</sup>;` once you confirm that.

mtruj013 replied:
Updated the href, will ask about the trademark

carkod replied:
I think hey [already replied](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit?disco=AAAAOf5RNKA)

mtruj013 replied:
Whoops, had not seen that, sorry! It's gone now :)

carkod commented on templates/about/release-cycle.html:
```diff
@@ -653,7 +653,7 @@ <h2 id="charmed-kubernetes-release-cycle">Charmed Kubernetes<sup>&reg;</sup> rel
     </div>
     <div class="row">
-      <p>For more information on previous and current releases of Charmed Kubernetes, please see the <a href="/kubernetes/docs/release-notes">Charmed Kubernetes release notes</a>.</p>
+      <p>For more information on previous and current releases, please see the <a href="https://ubuntu.com/kubernetes/docs/release-notes">Charmed Kubernetes</a> release notes or the <a href="https://microk8s.io/docs/release-notes">MicroK8s release notes.</a></p>
```
Same here, can you change it to relative path `/kubernetes/docs/release-notes`?
And the microk8s link is an external link so please add the `class="p-link--external"`

mtruj013 replied:
Done!

carkod commented:
Also, in the [copy docs, Nikos](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit?disco=AAAAIlDkPno) requested to remove the line 324 `5.4 kernel` which is not showing in the copy docs

mtruj013 commented:
> * Also, in the [copy docs, Nikos](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit?disco=AAAAIlDkPno) requested to remove the line 324 `5.4 kernel` which is not showing in the copy docs
> * The link to "Charmed Kubernetes" in the Contents section of Version Numbers needs to be updated too. I added it to the copy docs

Deleted the line and updated the link!
